src/pages/Learn/Quiz.jsx

```diff
@@ -26,16 +26,15 @@ const Quiz = () => {
       }
     } catch (error) {
       console.error('Error fetching quiz:', error);
+      navigate('/');
```

Comment:
Maybe navigate to the next lesson if possible, or course page now that it is available. Again up to you.